demos/python_demos/rag_chatbot/servable_stream/model.py

```diff
@@ -74,21 +70,6 @@ def __call__(
                 return True
         return False
 
-if stop_tokens is not None:
-    if isinstance(stop_tokens[0], str):
-        stop_tokens = tok.convert_tokens_to_ids(stop_tokens)
-    stop_tokens = [StopOnTokens(stop_tokens)]
-
-from ov_llm_model import model_classes
-model_class = (
```

Comment:
Did you test all 3 models (including notus and large llama) if those work after removing ov_llm_model?

Reply:
it would not be needed after updating optimum and refreshing. assuming only scope of 3 models it could be also dropped. It would be potentially a separate change.